Dockerfile

```diff
@@ -25,6 +25,7 @@ COPY cmd/manager/ cmd/manager/
 COPY controllers/ controllers/
 COPY mtls/ mtls/
 COPY runner/ runner/
+COPY internal/ internal/
```

Comment:
Why does the controller's Dockerfile need that directory. Not that it matters much.